app/views/admins/show_bills.html.erb

```diff
@@ -0,0 +1,12 @@
+All Bills | <%= link_to "Back To Admin Page", admins_path() %>
+
+<div>
+  <% @bills.each do |bill|%>
+    <div>
+    <%= bill.date %> <%= bill.price %> <%= bill.facility_id %> <%= bill.procedure_id %> <%= bill.status %>
+    <%= link_to "Edit", edit_bill_path(bill.id) %>
+    <%= link_to "Delete", bill_path(bill.id), method: :'delete' %>
```

Comment:
You don't need to use both a string and a symbol.

```suggestion
    <%= link_to "Delete", bill_path(bill.id), method: :delete %>
```

Reply:
changed to just a symbol.